site/config-overrides.md

```diff
@@ -138,34 +138,34 @@ _`wldf-MODULENAME.xml`_
 <wldf-resource>
 ```
```

Comment:
It would help, but I chose not to because of time constraints.   Ideally, we need a working .xml that demonstrates overriding a "Foreign JMS" URL, username, and password.     We'll very likely add something at a later time in a point release.